<!-- File: introduction.html -->
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <link rel="stylesheet" href="assets/css/styles.css" />
  <link href='https://cdn.jsdelivr.net/npm/boxicons@2.0.5/css/boxicons.min.css' rel='stylesheet'>
  <title>Introduction - Annelise Thorn</title>
</head>
<body>
  <!--===== HEADER NAVIGATION =====-->
  <header class="l-header">
    <nav class="nav bd-grid">
      <div><a href="index.html" class="nav__logo">Annelise Thorn</a></div>
    
      <div class="dropdown">
        <button class="dropbtn">Menu <i class='bx bx-chevron-down'></i></button>
        <div class="dropdown-content">
          <a href="introduction.html">Introduction</a>
          <a href="dataprep_eda.html">Data Prep/EDA</a>
          <a href="clustering.html">Clustering</a>
          <a href="pca.html">PCA</a>
          <a href="naivebayes.html">Naive Bayes</a>
          <a href="dectrees.html">Decision Tree</a>
          <a href="svms.html">SVM</a>
          <a href="regression.html">Regression</a>
          <a href="nn.html">Neural Net</a>
          <a href="conclusions.html">Conclusion</a>
          <a href="aboutme.html">About Me</a>
          <a href="references.html">References</a>
        </div>
      </div>
    </nav>
  </header>

  <main class="l-main">
    <section class="section" id="Introduction">
      <h2 class="section-title">Introduction</h2>
      <div class="introduction__container bd-grid">
        <h3>Topic: Predicting the Pathogenicity of Genetic Variants</h3>
        <p>“Almost every human trait and disease has a genetic component” (Genetic Alliance, 2009). 
          Every living organism is made up of cells, which contain DNA. 
          DNA acts as a set of instructions for all of the activities in the organism. 
          In human cells, DNA is packaged in larger units called chromosomes. 
          Each human typically contains 23 pairs of chromosomes; one pair of which are sex chromosomes (XX or XY). 
          Each chromosome contains segments of DNA, known as genes. 
          Genes encode for proteins and other functional molecules to carry out biological processes. 
          Small changes in DNA are referred to as genetic variants or mutations. 
          Some variants are harmless, while others can cause disease. 
          Understanding which genetic variants are harmful is crucial for the diagnosis and prevention of diseases. 
          Everyone has genetic variants, such as variations in eye color. 
          However, not everyone has pathogenic genetic variants.</p>
        <div style="display: flex; flex-direction: column; align-items: center; margin: 2rem 0;">
          <img src="assets/img/1.jpg" alt="Illustrative image for introduction" style="width: 50%; height: auto;" />
          <p style="margin-top: 0.5rem; font-size: 0.875rem; color: #555;">Figure 1: Add caption</p>
        </div>
        <p>“A typical human genome contains roughly 4 to 5 million single nucleotide variants (SNVs) and around 2,100 to 2,500 structural variants, such as large deletions or insertions” (Sabeti, 1970). 
          One type of structural variant is tandem repeats, which is when one or more nucleotides are repeated adjacent. 
          Several tandem repeat genetic variants are known to cause disease, like Huntington’s Disease and Fragile X Syndrome. 
          However, several tandem repeats have not been claimed as harmless or harmful; scientists call these variants of unknown significance (VUS). 
          VUSs can be frustrating for doctors and patients as doctors are left unsure how to treat and advise patients. 
          This uncertainty can cause anxiety, delays in treatment, or even incorrect diagnoses. 
          Additionally, different labs may interpret VUSs differently, leading to confusion for doctors and patients. 
          To help patients get clear results and better treatment options, it is essential to create better tools to understand VUSs. 
          Predicting which variants are harmful can play a critical role in medical genomics today. </p>
        <p>This problem is particularly important, because VUSs have serious consequences for patients and the healthcare system.
          When pathogenic or likely pathogeic variants are identified, patients can receive early diagnoses, appropriate medical care, and preventative measures.
          For example, if a healthy individual is found to have a pathogenic variant, like BRCA1, they can take proactive measures to monitor their health and reduce their risk of breast and ovarian cancer.
          On the other hand, if a harmful variant is misclassified or not understood, the opportunity for early intervention is lost and patients may face unnecessary risks.
          Not only is this a burden on patients, but it also places a strain on healthcare systems.
          Misclassifying variants can lead to misdiagnoses, inappropriate treatments, and increased healthcare costs.
          Additionally, understanding the pathogenicity of genetic variants can help researchers identify new disease mechanisms and develop targeted therapies.
          By accurately predicting the pathogenicity of genetic variants, we can improve patient outcomes, reduce healthcare costs, and advance the field of medical genomics.
          As genetic testing becomes more accessible and affordable, the need for accurate predictions of variant pathogenicity will only grow.
          Without being able to accurately predict the pathogenicity of genetic variants, the full benefits of genetic testing cannot be realized.
        </p>
        <p>Paragraph 4- What has been done before</p>
        <p>Paragraph 5- What can be done now</p>
        <h3>Research Questions</h3>
          <ol>
            <li>Can a machine learning model determine whether a genetic variant is harmful?</li>
            <!-- ... -->
            <li>Which features, such as variant frequency or genomic location, are most helpful in predicting whether a variant is harmful?</li>
            <!-- ... -->
            <li>Which machine learning model tends to perform best for this kind of prediction?</li>
            <!-- ... -->
            <li>Does using more features generally lead to more accurate predictions?</li>
            <!-- ... -->
            <li>Can a model trained on one population group still perform well when applied to a different group?</li>
            <!-- ... -->
            <li>Does including conservation scores improve the accuracy of the model’s predictions?</li>
            <!-- ... -->
            <li>Which features does the model find most important when making its predictions?</li>
            <!-- ... -->
            <li>Can genetic variants be grouped into clusters based on their shared features?</li>
            <!-- ... -->
            <li>How well does the model perform when some of the feature data is missing or noisy?</li>
            <!-- ... -->
            <li>Does the model perform better on certain types of variants, like missense compared to synonymous?</li>
            <!-- ... -->
          </ol>
      </div>
    </section>
  </main>

  <script src="https://unpkg.com/scrollreveal"></script>
  <script src="assets/js/main.js"></script>
</body>
</html>
